core/src/main/java/org/eclipse/hono/util/DeviceManagementConstants.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.hono.util;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Constants &amp; utility methods used throughout the Device Management API.
+ */
+public final class DeviceManagementConstants extends RequestResponseApiConstants {
```

Comment:
this seems to be the place where I would like to see *all* constants relevant for the HTTP management API being defined ...

Reply:
I tried to consolidate all constants here (and renamed it `RegistryManagementConstants` (better name ideas?). Note, that file based registry implementation will still use AMQP API constants and objects where applicable as underlying data structure is optmized for that.